src/automation/tasks/scripts/guidellm_script.py

```diff
@@ -1,35 +1,46 @@
-
 import os
+import sys
 from clearml import Task
 from automation.utils import resolve_model_id, cast_args, kill_process_tree
 from automation.vllm import start_vllm_server
 from pyhocon import ConfigFactory
-
+from automation.configs import DEFAULT_GUIDELLM_SCENARIO
 
 def main(configurations=None):
     task = Task.current_task()
 
     args = task.get_parameters_as_dict(cast=True)
     
-    if configurations is None:
-        guidellm_args = ConfigFactory.parse_string(task.get_configuration_object("GuideLLM"))
-    
-        environment_args = task.get_configuration_object("environment")
-        if environment_args is None:
-            environment_args = {}
-        else:
-            environment_args = ConfigFactory.parse_string(environment_args)
+    raw_config = task.get_configuration_object("GuideLLM")
```

Comment:
This logic for configurations is flawed. It only works when running remotely, not when running locally. When running locally the process doesn't have access to configuration objects